### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by rpinoon**
So I'm trying to get into handling errors from the api request. Since these errors aren't going to be used by any other component, I decided to just show an error message through Toastfiy.

Please let me know how I can improve this

### CODE REVIEW

File: api/utils.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ToastMessage } from "components/toast"
+
+export interface genericError {
+  message: string
+  request: { status: number }
+}
+
+export const errorHandler = (error: genericError) => {
+  switch (error.request.status) {
+    case 0:
```
Comment (mikodagatan) [inline api/utils.ts:11]: put `ToastMessage({ type: "error", message: message })` outside of the case,

only put the `message` in the case

Comment (rpinoon) [reply api/utils.ts:11]: good idea. copy that

Comment (rpinoon) [reply api/utils.ts:11]: is it done like this?
image

Comment (mikodagatan) [reply api/utils.ts:11]: Yeah

**Pull request merged**